src/js/form-render.js

```diff
@@ -82,10 +82,34 @@ class FormRender {
           className: 'rendered-form',
         })
         this.appendChild(renderedFormWrap)
+  
         fields.forEach(field => {
-          renderedFormWrap.appendChild(field)
-          field.dispatchEvent(events.fieldRendered)
+          // Determine if rows are being used. If so, create the row and append to its row-{group}
+          // If the fields have row-, create & append to the appropriate row
+          const classes = field.className.split('row-');
+          if(classes[1]){           
+            const rowGroup = classes[1].split(' ')[0];
+            const rowID = this.id ? `${this.id}-row-${rowGroup}` : `row-${rowGroup}`;
+
+            // Check if this rowID is created yet or not.            
+            let rowGroupNode = document.getElementById(rowID);
```

Comment:
Can use `utils.markup` and avoid if block on [L96](https://github.com/kevinchappell/formBuilder/pull/859/files#diff-47035f04c42bc9d79ae2e3e19a6eb04fR96)
```suggestion
            const rowGroupNode = document.getElementById(rowID) || utils.markup('div', null, { id: rowID, className: 'row form-inline' });
```

Reply:
Can't immediately apply the markup because as this keeps looping I need to check if the row is created or not for each field found. I can use utils to create the markup once I find it is not created.